temp/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Please disregard this directory, it is used to store Modmail temp files.
+# DO NOT REMOVE THIS DIRECTORY.
+*
+!.gitignore
```

Comment:
why don't you just git ignore the whole dir, and create it on bot start?

Reply:
True, also a valid option.

Reply:
Regardless, it's created on start now.